content/docs/secure/encryption/rekey.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Rekey Configuration"
+linkTitle: "Rekey Configuration"
+weight: 3

```

Comment:
Please update weight for all pages:
deplyment - 1
uninstallation - 2
vault - 3
rekey - 4
troubleshooting -5
release - 6

Reply:
updated